src/axom/quest/detail/MarchingCubesPartParallel.hpp

```diff
@@ -45,9 +36,19 @@ static void add_to_StackArray(axom::StackArray<T, DIM>& a, U b)
   axom::CUDA_EXEC<256>.  SequentialExecSpace is used for loops that
   cannot be parallelized but must access data allocated for ExecSpace.
   Use something like axom::SEQ_EXEC or axom::CUDA_EXEC<1>.
+
+  The difference between MarchingCubesPartParallel and MarchingCubesFullParallel
+  is how they compute indices for the unstructured surface mesh.
+  - MarchingCubesPartParallel uses sequential loop, and skips over parents
+    cells that don't touch the surface contour.  It processes less data
+    but is not data-parallel.
+  - MarchingCubesFullParallel that checks all parents cells.
+    It process more data but is data-parallel.
+  We've observed that MarchingCubesPartParallel is faster for seq policy
+  and MarchingCubesPartParallel is faster on the GPU.
 */
 template <int DIM, typename ExecSpace, typename SequentialExecSpace>
-class MarchingCubesImpl : public MarchingCubesSingleDomain::ImplBase
+class MarchingCubesPartParallel : public MarchingCubesSingleDomain::ImplBase
```

Comment:
Naming: `Part` doesn't necessarily read as "partial" (to me). 

Would "Hybrid" work better?

Reply:
I like "hybrid".  Thanks!